modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -2,16 +2,27 @@
 
 :link-vector-indexes: xref:indexes/semantic-indexes/vector-indexes.adoc
 
-[role=on-aura]
 [[genai-integrations]]
 = GenAI integrations
 
-_This feature is available only in Aura._
+_This feature is available from Neo4j 5.17 in both Neo4j Aura and on-premises deployments.
 
 Neo4j provides integrations with various generative AI services using a native GenAI plugin.
 This includes support for transforming text data into vector embeddings using VertexAI, OpenAI, or Amazon Bedrock.
 The resulting vector embeddings can be used together with Neo4j's {link-vector-indexes}[vector search indexes].
 
+== Prerequisites
+
+To use the GenAI plugin, you need to have the following:
+
+* A Neo4j DBMS running Neo4j 5.17 or later.
+* A GenAI provider account, such as VertexAI, OpenAI, or Amazon Bedrock.
+* API credentials for the provider you want to use.
+* (On-prem only) the Neo4j GenAI plugin installed in your Neo4j deployment.
+The plugin JAR is located in the _/product_ directory of the Neo4j installation.
```

Comment:
This is a bit of a nitpick but is this how usually we format relative paths in the docs? Normally a leading `/` would imply an absolute path. The meaning is easily inferred from the context of course; I'm just double checking. 🙂

Reply:
Fixed.